### ISSUE TITLE
[BUG] Correcting Video Date - Original date persists because of transcoded video

### ISSUE THREAD

**Issue by PancakeZik** [immich-app/immich#4368]
### The bug

Hey everyone,
I imported my external libraries into Immich, and I had a lot of photos and videos with the wrong dates.
They were mostly grouped in the same date.
I corrected most of them. For photos, after refreshing metadata, it got the correct date and went to the correct position in the timeline.
However, for videos, I noticed something strange:
Even after getting the correct date, it would stay in the same position in the timeline.
After investigating, I found out that was only happening to videos which were transcoded: Immich was reading the new date, but to place it in the timeline it was still using the date for the original transcoded video, which wasn't corrected.

### The OS that Immich Server is running on

Docker

### Version of Immich Server

1.8.1

### Version of Immich Mobile App

Web

### Platform with the issue

- [X] Server
- [ ] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
Default compose
```

### Your .env content

```Shell
Default Env
```

### Reproduction steps

```bash
1. Add video with wrong file created and modified date to Immich
2. Let Immich transcode it
3. Correct the original file date
4. Refresh metadata for the item
5. Date is correct, but it's still placed in the wrong position
```

**Comment by alextran1502** [comment]
Hello, how did you correct the metadata?

**Comment by PancakeZik** [comment]
For the photos I used EXIF Date changer, as well as changing the created date.
For the videos I changed the Created and Modified date using some simple Windows utilities.

**Comment by alextran1502** [comment]
I assume that the keys you changed aren't supported. Do you mind sharing the troublefile?

**Comment by PancakeZik** [comment]
I'll close this, since I realised there is a big difference between .mp4 and .mpg files.
Maybe the issue with the .mp4 wasn't the transcoding, but actually it got stuck with the wrong date before the fixes in v1.8
